modules/microshift-embed-kickstart-in-iso.adoc

```diff
@@ -1,40 +1,29 @@
 // Module included in the following assemblies:
 //
-// microshift/microshift-embed-into-rpm-ostree.adoc
+// microshift_install_rpm_ostree/microshift-embed-in-rpm-ostree.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="provisioning-a-machine_{context}"]
-= Provisioning a machine for {microshift-short}
+[id="microshift-embed-kickstart-iso_{context}"]
+= Embedding a Kickstart file in an ISO
 
-Provision a machine with your {op-system-ostree} image by using the procedures from the {op-system-ostree} documentation.
-
-To use {microshift-short}, you must provision the system so that it meets the following requirements:
-
-* The machine you are provisioning must meet the system requirements for installing {microshift-short}.
-* The file system must have a logical volume manager (LVM) volume group (VG) with sufficient capacity for the persistent volumes (PVs) of your workload.
-* A pull secret from the https://console.redhat.com/openshift/install/pull-secret[Red Hat Hybrid Cloud Console] must be present as `/etc/crio/openshift-pull-secret` and have root user-only read/write permissions.
-* The firewall must be configured with the required settings.
-
-[NOTE]
-====
-If you are using a Kickstart such as the {op-system-ostree} Installer (ISO) image, you can update your Kickstart file to meet the provisioning requirements.
-====
+You can use the Kickstart provided with {microshift-short}, or you can update an existing {op-system-ostree} Installer (ISO) Kickstart file. When ready, embed the Kickstart into the ISO.
 
 .Prerequisites
 
-. You have created a {op-system-ostree} Installer (ISO) image containing your {op-system-ostree} commit with {product-title}.
-.. This requirement includes the steps of composing an RFE Container image, creating the RFE Installer blueprint, starting the RFE container, and composing the RFE Installer image.
-. Create a Kickstart file or use an existing one. In the Kickstart file, you must include:
-.. Detailed instructions about how to create a user.
-.. How to fetch and deploy the {op-system-ostree} image.
+* You created a {op-system-ostree} Installer (ISO) image containing your {op-system-ostree} commit with {microshift-short}. This requirement includes the following steps:
+** Composing an {op-system-ostree} Container image
+** Creating the {op-system-ostree} Installer blueprint
+** Starting the {op-system-ostree} container
+** Composing the {op-system-ostree} Installer image.
```

Comment:
Let's remove these extra steps here. It's hard to give a precise list of what's required and the right order.

Reply:
It is step 2 https://85240--ocpdocs-pr.netlify.app/microshift/latest/microshift_install_kickstarts/microshift-rhel-kickstarts#microshift-kickstart-prep_microshift-rhel-kickstarts, @dhensel-rh.

Reply:
This has always been here, it is just reformatted. I do not think just saying, "You created a {op-system-ostree} Installer (ISO) image containing your {op-system-ostree} commit with {microshift-short}" is enough for a novice user. What kind of general guidance should we give instead, then?

Reply:
That's the general guidance. I do not think the steps help because there are many ways to create ISO images. We can refer the user to the appropriate section in the docs instead.